x-pack/plugins/cloud/server/config.ts

```diff
@@ -18,6 +18,16 @@ const apmConfigSchema = schema.object({
   ),
 });
 
+const fullStoryConfigSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: false }),
+  orgId: schema.conditional(
```

Comment:
should it be `org_id` for consistency with this plugin's other configuration keys?